### PR TITLE
Icp 7154: change device name to chinese

### PR DESCRIPTION
**Pull request by FenMei007S**
@tpmanley @varzac  Dear Tom and Zach, plz review it,and merage to china STG server. the device name should be changed to chinese

### CODE REVIEW

File: devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy [hunk 1]
```diff
@@ -39,7 +39,7 @@ metadata {
 		fingerprint inClusters: "0000,0001,0003,000F,0020,0402,0500", outClusters: "0019", manufacturer: "SmartThings", model: "motionv5", deviceJoinName: "Motion Sensor"
 		fingerprint inClusters: "0000,0001,0003,0020,0400,0500,0B05", outClusters: "0019", manufacturer: "Bosch", model: "RFPR-ZB", deviceJoinName: "Bosch Motion Sensor"
 		fingerprint inClusters: "0000,0001,0003,000F,0020,0402,0500", outClusters: "0019", manufacturer: "Bosch", model: "RFDL-ZB-MS", deviceJoinName: "Bosch Motion Sensor"
-		fingerprint inClusters: "0000,0001,0003,0020,0402,0500", outClusters: "0019", manufacturer: "Samjin", model: "motion", deviceJoinName: "Motion Sensor"
+		fingerprint inClusters: "0000,0001,0003,0020,0402,0500", outClusters: "0019", manufacturer: "Samjin", model: "motion", deviceJoinName: "人体传感器"
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy]: These join names are used globally so we cannot change the join name to Chinese for the SmartSense Motion Sensor or SmartSense Multi Sensor.

**Review by tpmanley — CHANGES_REQUESTED** [review]
I'm not opposed to changing the join names for the devices sold in the Chinese market but the names for the devices sold globally should not be changed.

**Comment by FenMei007S** [comment]
@ tpmanley, Is it possible only push the change to china server?

**Comment by scarlettw0913** [comment]
@tpmanley, we have verified chinese device names, only SDS doorlock device name still can not work, please help to check it, thanks~

**Comment by tpmanley** [comment]
@scarlettw0913 , I verified the changes had been deployed already so I'll need some more information about the problem. Please hit me up in Slack

**Comment by FenMei007S** [comment]
@tpmanley, Dear Tom, the switch's name change, it makes siwtch name incorrect, so update code to fix it.plz help to merage it to china server

**Comment by FenMei007S** [comment]
change separator for new device name

**Pull request closed without merging**